Resources/Prototypes/Maps/drydock.yml

```
@@ -0,0 +1,72 @@
+﻿- type: gameMap
+  id: DryDock
+  mapName: 'DryDock Station'
+  mapPath: /Maps/DryDock.yml
+  minPlayers: 10
+  maxPlayers: 50
+  stations:
+    DryDockStation:
+      stationProto: StandardNanotrasenStation
+      components:
+      - type: StationNameSetup
+        mapNameTemplate: '{0} DryDock Station {1}'
+        nameGenerator:
+          !type:NanotrasenNameGenerator
+          prefixCreator: 'WWDP'
+      - type: StationEmergencyShuttle
+        emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_Delta.yml
+      - type: StationJobs
+        overflowJobs:
+        - Passenger
+        availableJobs:
+          #Heads
+          Captain: [ 1, 1 ]
+          HeadOfSecurity: [ 1, 1 ]
+          HeadOfPersonnel: [ 1, 1 ]
+          ChiefEngineer: [ 1, 1 ]
+          ChiefMedicalOfficer: [ 1, 1 ]
+          ResearchDirector: [ 1, 1 ]
+          Quartermaster: [ 1, 1 ]
+          #Security
+          Warden: [ 1, 1 ]
+          SeniorOfficer: [ 1, 1 ]
+          SecurityOfficer: [ 4, 4 ]
+          SecurityCadet: [ 4, 4 ]
+          Detective: [ 1, 1 ]
+          Lawyer: [ 2, 2 ]
+          #Medical
+          SeniorPhysician: [ 1, 1 ]
```

Роль отсутствует раундстартом.
```suggestion
          #SeniorPhysician: [ 1, 1 ]
```